openspacenet/include/OpenSpaceNetArgs.h

```diff
@@ -0,0 +1,98 @@
+/********************************************************************************
+* Copyright 2017 DigitalGlobe, Inc.
+* Author: Joe White
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+********************************************************************************/
+
+#ifndef OPENSPACENET_OPENSPACENETARGS_H
+#define OPENSPACENET_OPENSPACENETARGS_H
+
+#include <vector/Feature.h>
+
+#define OSN_LOG(sev) DG_LOG(OpenSpaceNet, sev)
+#define MAPSAPI_MAPID  "digitalglobe.nal0g75k"
```

Comment:
Not a real review point, but we may want to consider moving this out to a loadable default config file, and not in code